src/main/java/storage/IntArrayStack.java

```diff
@@ -0,0 +1,52 @@
+package storage;
+
+import edu.kis.vh.nursery.Generate;
+
+public class IntArrayStack implements Generate {

```

Comment:
IntArrayStack should have common interface with IntLinkedList not Default...Ryhmer.